tradingstrategy/charting/candle_chart.py

```diff
@@ -314,4 +313,123 @@ def visualise_ohlcv(
             "x": 1,
         })
 
-    return fig
+def _get_volume_grid(
+    volume_bars, 
+    volume_bar_mode: bool, 
+    volume_axis_name: str, 
+    num_detached_indicators: int,
+    vertical_spacing: float,
+    relative_sizing: list[float],
+    subplot_names: list[str]
+) -> go.Figure:
+    """Get subplot grid, with volume information, based on the volume bar mode"""
+    
+    is_secondary_y = _get_secondary_y(volume_bar_mode)
+    
+    if relative_sizing and not all(relative_sizing):
+        relative_sizing = None
+    
+    # blank subplot name for the main price chart
+    if subplot_names:
+        subplot_names.insert(0, None)
+
+    if volume_bar_mode == VolumeBarMode.separate:
+        # If separate, we need to use detached subplots
+        
+        if relative_sizing:
+            row_heights = [PRICE_CHART_REL_SIZE] + [DEFAULT_REL_SIZE] + relative_sizing
```

Comment:
Do not use module constants.

Because if the user wants to change this value, they would need to fork the project.

Instead, pass everything as parameters.